docs/content/docs/how-to/add-handwritten-datasource-documentation.md

```diff
@@ -0,0 +1,157 @@
+That page contains information about how documentation should be structured and how you can test changes to documentation.
+
+This page includes only instructions on how to add documentation for a new handwritten data source, with minimal background info.
+
+## Finding the relevant file location
+
+Handwritten documentation is located in the `website/docs` folder, shown below.
+
+```
+mmv1/third_party/terraform/website/docs/
+├─ guides/
+│  ├─ ...
+├─ d/
+│  ├─ ...
+├─ r/
+│  ├─ ...
+├─ index.html.markdown
+```
+
+The subfolder `d` corresponds to data sources, and inside there is a file for each data source.
+
+## Creating the new markdown file
+
+Next you need to add the file for the new data source's documentation. Create a new file inside the `d` folder. The filename should be the name of the data source with the `google_` prefix removed. For example if you were adding a new data source `google_foobar` then you would need to create a new file with the name `foobar.html.markdown`.
+
+## Adding contents
+
+Pages within the documentation need to be consistent and contain the sections that users expect. Below is some guidance about the different sections to include and what their contents should be.
+
+### Frontmatter
+
+The top of the file needs to contain frontmatter, which is used to create the new documentation page's title and manage how the page is linked to in the documentation's sidebar navigation. It it not rendered but it is very important for the new page in the documentation to be generated and available to users.
+
+You need to make sure your file's frontmatter includes:
+- `subcategory` - This sets which section in the sidebar navigation (left side) the new page is categorised into.
+- `page_title` - This frontmatter is specific to files in the Guides section (`/website/docs/guides`) but we set it for all markdown files.
+- `description` - A decription of the page.
+
+For example, here is the frontmatter from `/website/docs/d/container_cluster.html.markdown` ([link to generated page](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/container_cluster)):
+
+```markdown
+---
+subcategory: "Kubernetes (Container) Engine"
+page_title: "Google: google_container_cluster"
+description: |-
+  Get info about a Google Kubernetes Engine cluster.
+---
+```
+
+### Page title and description
+
+The next section in the markdown file is rendered as the first part of the page body.
+
+It should contain:
+- the page title, as an H1 header
+- a description general information about the data source
+
+The description can be as long or as short as necessary. The minimum information that's included in this section are links to official documentation and the API reference pages. Other guidance, warnings, or explanations of concepts can be included here. To create pronounced warning or info sections, see the [Documentation](#documentation) page for info.
```

Comment:
Replace `[Documentation](#documentation)`  with link to `getting-started/provider-documentation.md`